homeassistant/components/google_generative_ai_conversation/__init__.py

```diff
@@ -97,11 +97,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     genai.configure(api_key=entry.data[CONF_API_KEY])
 
     try:
-        await hass.async_add_executor_job(
-            partial(
-                genai.get_model, entry.options.get(CONF_CHAT_MODEL, DEFAULT_CHAT_MODEL)
-            )
-        )
+        await hass.async_add_executor_job(partial(genai.get_model, DEFAULT_CHAT_MODEL))
```

Comment:
I changed this because when I had configured a bad model, the config entry refused to startup

Reply:
Why is that bad? Isn't it better to fail as soon as possible?

Reply:
The config entry wouldn't set up and the user couldn't change the model anymore 🙈

Reply:
(happened to me)